### PR TITLE
RSDK-5282: Obey runtime changing of component configuration log levels.

### PR DESCRIPTION
**Pull request by dgottlieb**

### CODE REVIEW

File: logging/logger.go [hunk 1]
```diff
@@ -47,12 +48,6 @@ type ZapCompatibleLogger interface {
 	Fatalw(msg string, keysAndValues ...interface{})
 }
 
-// zLogger type for logging to. Wraps a zap logger and adds the `AsZap` method to satisfy the
```
Comment (dgottlieb) [inline logging/logger.go:50]: Didn't intend to move this. I just merge-conflicted poorly (I had this patch created prior to turning on all logging).

File: logging/logger.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ type Logger interface {
 	ZapCompatibleLogger
 
 	SetLevel(level Level)
+	GetLevel() Level
```
Comment (dgottlieb) [inline logging/logger.go:13]: Not strictly necessary (`ZapCompatibleLogger` has a `Level() zapcore.Level`). But I had some printlns and the mixing of cases in `zapcore.Level.String` and `logging.Level.String` bothered me.

**Review by stuqdog — APPROVED** [review]
lgtm!

File: logging/logger.go [hunk 3]
```diff
@@ -73,13 +68,21 @@ func FromZapCompatible(logger ZapCompatibleLogger) Logger {
...
+
 func (logger *zLogger) SetLevel(level Level) {
 	// Not supported
 }
 
-func (logger *zLogger) Sublogger(subname string) Logger {
+func (logger *zLogger) GetLevel() Level {
 	// Not supported
-	return nil
+	return INFO
```
[the remark this answers is not in the document]
Comment (dgottlieb) [reply logging/logger.go:85]: It's not a shift, just a poor semantic diff. `nil` was for `Sublogger` (which is an interface type, so nil is available). Level is a structure type.

**Pull request merged**